ci-operator/step-registry/upi/install/aws/cluster/upi-install-aws-cluster-commands.sh

```diff
@@ -301,6 +320,7 @@ add_param_to_json MasterInstanceType "${MASTER_INSTANCE_TYPE}" "${cf_params_cont
 
 cat "${cf_params_control_plane}"
 
+echo "${CLUSTER_NAME}-control-plane" >> "${NEW_STACKS}"
```

Comment:
How about introduce a var to save `"${CLUSTER_NAME}-control-plane"`, so that we can use the var in this command and the next command.